ecosystem/indexer-grpc/indexer-grpc-cache-worker/src/worker.rs

```diff
@@ -158,9 +170,19 @@ async fn process_raw_datastream_response(
             match cache_operator.update_cache_transactions(transactions).await {
                 Ok(_) => {},
                 Err(e) => {
+                    ERROR_COUNT
+                        .with_label_values(&["failed_to_update_cache_version"])
+                        .inc();
                     anyhow::bail!("Update cache with version failed: {}", e);
                 },
             }
+            if let Some(txn_time) = first_transaction_timestamp_in_secs {
```

Comment:
Could we use the helper function you made `time_diff_since_pb_timestamp_in_secs`?

Reply:
done. good point.